package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@influxdata/clockface",
-  "version": "6.0.2",
+  "version": "6.0.3",
```

Comment:
minor nitpick, but this might be considered new, non-breaking functionality, which means this could be `6.1.0`